test/onnx/verify.py

```diff
@@ -193,7 +195,7 @@ def __exit__(self, exc_type, exc_value, traceback):
                     return True
         return Recover()
 
-    def addErrCtxt(parent_self, msg):
+    def addErrCtxt(self, msg):
```

Comment:
Could you replace references to `parent_self` with `self` and remove `parent_self` ?

Reply:
No, because as per the lint rule, the inner class definition also has to use self for methods, and that results in shadowing :)